CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+The `.vscode` folder contains launch configurations so you can test the extension.
+
+### Get started
+
+- Clone the repo
+- `yarn install`
+- Open VSCode and run the `Launch Extension` target
+
+### Tests
+
+Tests need to be run by the VSCode extension host, so they are provided by a launch configuration as well.
+
+They then run as a Mocha test suite in the extension host, so their output isn't usually visible.
+
+### Design considerations
+
+The new fix to allow for multi-root workspaces means that the search criteria apply to all workspace folders at once.
```

Comment:
Because "new fix" is a relative term, how would you feel about something like this: 

```suggestion
Based on the nature of multi-root workspaces and how they operate, it means that the search criteria apply to all workspace folders at once.
```